content/api/commands/session.md

```diff
@@ -944,5 +944,5 @@ data, including cookies, `localStorage` and `sessionStorage`.
 - [Authenticate faster in tests with the cy.session command](https://cypress.io/blog/2021/08/04/authenticate-faster-in-tests-cy-session-command/)
```

Comment:
Looks like there's another area where we can flip the order of these commands at /api/commands/session#Common-Questions